src/streaming/transcoders.ts

```diff
@@ -0,0 +1,322 @@
+    }
+
+    const queryOptions = {
+        hostname: Urls.getRtsHostname(),
+        path: '/api/transcoders',
+        params,
+        headers: {
+            Accept: 'application/json',
+            Authorization: `Bearer ${apiSecret}`,
+        },
+    };
+
+    return await sendGet<Transcoder[]>(queryOptions);
+};
+
+/**
+ * Gets the specified cloud transcoder.
+ *
+ * @link https://docs.dolby.io/streaming-apis/reference/transcoder_gettranscoder
+ * @beta
+ * @remarks Cloud transcoder usage is not currently available for general usage.
+ * If you would like to opt in, please contact our Sales team.
+ *
+ * @param apiSecret The API Secret used to authenticate this request.
+ * @param transcoderId Identifier of the cloud transcoder to retrieve.
+ *
+ * @returns A {@link !Promise Promise} whose fulfillment handler receives a {@link Transcoder} object.
+ */
+export const getTranscoder = async (apiSecret: string, transcoderId: string): Promise<Transcoder> => {
+    const options = {
+        hostname: Urls.getRtsHostname(),
+        path: `/api/transcoders/${transcoderId}`,
+        headers: {
+            Accept: 'application/json',
+            Authorization: `Bearer ${apiSecret}`,
+        },
+    };
+
+    return await sendGet<Transcoder>(options);
+};
+
+/**
+ * Deletes the specified cloud transcoder.
+ *
+ * @link https://docs.dolby.io/streaming-apis/reference/transcoder_deletetranscoder
+ * @beta
+ * @remarks Cloud transcoder usage is not currently available for general usage.
+ * If you would like to opt in, please contact our Sales team.
+ *
+ * @param apiSecret The API Secret used to authenticate this request.
+ * @param transcoderId Identifier of the cloud transcoder to delete.
+ *
+ * @returns A {@link !Promise Promise} whose fulfillment handler receives a flag to indicate if the operation succeeded or not.
+ */
+export const deleteTranscoder = async (apiSecret: string, transcoderId: number): Promise<boolean> => {
+    const options = {
+        hostname: Urls.getRtsHostname(),
+        path: `/api/transcoders/${transcoderId}`,
+        headers: {
+            Accept: 'application/json',
+            Authorization: `Bearer ${apiSecret}`,
+        },
+    };
+
+    return await sendDelete<boolean>(options);
+};
+
+/**
+ * Creates a cloud transcoder.
+ *
+ * @link https://docs.dolby.io/streaming-apis/reference/publishtokenv1_createtoken
+ * @beta
+ * @remarks Cloud transcoder usage is not currently available for general usage.
+ * If you would like to opt in, please contact our Sales team.
+ *
+ * @param apiSecret The API Secret used to authenticate this request.
+ * @param publishToken Information about the new publish token.
+ *
+ * @returns A {@link !Promise Promise} whose fulfillment handler receives a {@link Transcoder} object.
+ */
+export const createTranscoder = async (apiSecret: string, transcoder: CreateTranscoderRequest): Promise<Transcoder> => {
+    if (transcoder.startNow == null) {
```

Comment:
`startNow` and `passThrough` are optional parameters, but is **false** the default for `startNow`?

Reply:
It should be `false`, I'm changing it.